Added new font, background etc

al3xandra33 opened the pull request.

al3xandra33 committed:
Added new font, background etc

anjali-kr commented on app/views/shared/_navbar.html.erb:
```diff
@@ -1,12 +1,12 @@
     <form class="d-flex align-items-center flex-nowrap" action="/offers" method="get">
 
-    <input class="form-control mr-sm-2" name ="query" type="search" placeholder="Find a cooking class" aria-label="Search">
+    <input class="form-control mr-sm-2" name ="query" type="search" placeholder="Find a chef" aria-label="Search">
```
We could go with a more generic message here, to indicate that a user can search for a chef or a cuisine or a food.

al3xandra33 replied:
The space is very small so we cannot add a list. If you have any suggestions, you can write here.  Anyways, it is a very small detail.

anjali-kr replied:
Not suggesting a list. I meant a more generic message like 'What are you looking for?' or 'Find your next culinary experience'.. Something along those lines. It is a small detail but one that appears on every screen of the website.

We could merge this for now, then discuss and change on Saturday.

anjali-kr commented on app/assets/stylesheets/config/_bootstrap_variables.scss:
```diff
@@ -23,3 +23,4 @@ $border-radius-lg: 2px;
 $border-radius-sm: 2px;
 
 // Override other variables below!
+body { background: #ffffff !important;}
```
The recommended way of doing this would be to add a new color in config/colors and then changing body-bg in this file.

al3xandra33 replied:
Is there a reason why that is the recommended way? this way works just fine ...

anjali-kr replied:
The two links at the top of this file (in the comments) explain how scss  variables are declared and overriden. 

In this case you're basically overriding body-background colour 2 times, first on line 9 and  then again on line 26.

anjali-kr replied:
Looking at the bootstrap variables again, I realised that the default bootstrap body-bg color is already white. So in this case, we don't need to override anything. Simply commenting out line 9 will work.

johanna-maier replied:
I will try Anjali suggestion and merge

al3xandra33 committed:
Update _navbar.html.erb

johanna-maier committed:
Merge branch 'master' into small-design-modifications

johanna-maier committed:
Updated bg-color to bootstrap default (white)

Commented out: // $body-bg:                 $light-gray;
Not needed anymore, so commented out too: // { background: #ffffff !important;}
